pkg/process/util/containers_test.go

```diff
@@ -485,12 +499,12 @@ func TestGetContainers(t *testing.T) {
 			TotalPct:    -1,
 			MemRss:      42000,
 			MemCache:    200,
-			Rbps:        20,
-			Wbps:        40,
-			NetRcvdPs:   42,
-			NetSentPs:   42,
-			NetRcvdBps:  4.3,
-			NetSentBps:  4.2,
+			Rbps:        0,
+			Wbps:        0,
+			NetRcvdPs:   0,
+			NetSentPs:   0,
+			NetRcvdBps:  0,
+			NetSentBps:  0,
```

Comment:
Just to better understand it, why have these tests values changed?

Reply:
Previously we were relying on `lastRun` shared timestamp to decide if we should compute rates or not. Now the timestamp is per container. As we delete the container from the `lastRate` struct, we don't have previous timestamp anymore and thus values are not computed

Reply:
Gotcha. Thanks for clarifying it!